Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN \
     rm -Rf target node_modules && \
     chmod +x /code/mvnw && \
     sleep 1 && \
-    ./mvnw package -Pprod -DskipTests && \
+    ./mvnw package -Pprod -Pgcp -DskipTests && \
```

Comment:
If we are using this docker file for exclusive gcp deployments, then, I suggest renaming to reflect the context. Further, the maven `gcp` profile includes the spring `prod` profile.

Reply:
You are correct; I've changed it to only include the gcp profile. As for renaming the docker file; currently that's not possible. Because, `gcloud builds submit` command expects the file to be named as `Dockerfile`. :cry: